sys/linux/vnet_386.const

```diff
@@ -339,6 +339,21 @@ NEXTHDR_ICMP = 58
 NEXTHDR_MOBILITY = 135
 NEXTHDR_NONE = 59
 NEXTHDR_ROUTING = 43
+ND_OPT_SOURCE_LL_ADDR = 1
```

Comment:
These constants are all just a single byte, so they'll be the same for all architectures regardless of endianness. Might be nice to add a `vnet_any.const` for constants like these?

Reply:
These files are all auto-generated, so there is no cost in that duplication.

Reply:
:+1: